isamples_metadata/SESARTransformer.py

```diff
@@ -36,17 +41,80 @@ def sample_description(self) -> typing.AnyStr:
         # TODO: implement
         return Transformer.NOT_PROVIDED
 
-    def has_context_categories(self) -> typing.List:
+    def has_context_categories(self) -> typing.List[ContextCategory]:
+        # From Steve: hasContextCategory (SampledFeatureType) This one’s more complicated, here’s a start at the logic:
+        # is it an Ocean Drilling Program sample (assume its core from the ocean floor) or is the material rock or a subtype of rock or is the sampleType ‘Core’; hasContextCategory is RockBody
+        # is material type coral, (ideally verify that its from a living coral species—check field name and geologic age): hasContextCategory is MarineBiome
+        # Is platform type a submersible (e.g. Alvin)?  If Material type is rock or gas hasContextCategory is ‘Marine water body bottom’
+        # primaryLocationType is good, but only populated in 268,454 records;
```

Comment:
Can mostly ignore these comments, these are placeholders for our discussion tomorrow.